main/settings.py

```diff
@@ -282,6 +282,7 @@
 ]
 
 TINYMCE_DEFAULT_CONFIG = {
+    'images_upload_url': '/upload_image',  # Image upload address in Django route
```

Comment:
Maybe we can use reverse_lazy.
https://docs.djangoproject.com/en/3.2/ref/urlresolvers/#reverse-lazy

Reply:
I was not able to get this reverse_lazy work. Please give me some more hints.
I gave a name in urls.py, but there are other tasks also.
```
    url(r"^upload_image", upload_image, name="upload_image"),
```

Reply:
Hey @szabozoltan69 Did you mean there are other URLs with the same name? Maybe we can use a different name instead? Like "tinymc_upload_image"
Let me know if I understood incorrectly.

Reply:
Hey @thenav56 , there is no other same name, just for the sake of reverse_lazy I gave a name to the endpoint. But I could not do the next steps about it. Maybe not even meeded to be named.

Reply:
As "other tasks" I meant other developing tasks for sake of reverse_lazy.

Reply:
Okay. Let's leave with a TODO there then.